pkg/rancher-desktop/assets/scripts/wsl-exec

```diff
@@ -16,6 +16,15 @@ if [ -z "${pid}" ]; then
     exit 1
 fi
 
+# If the pid is _not_ /sbin/init, find the child that is.
+command="$(ps -o pid,args | awk "\$1 == $pid { print \$2 }")"
+if [ "$command" != "/sbin/init" ]; then
+  newpid="$(ps -o pid,ppid,args | awk "\$2 == $pid && \$3 == \"/sbin/init\" { print \$1 }")"
```

Comment:
Would it make sense to do `pgrep -f /sbin/init` instead?

Reply:
I want to make sure if there's multiple instances of `/sbin/init` (… somehow) we find the correct one.